src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -70,24 +76,43 @@ const Index = ({
     preselectedProgramId,
     searchStatus,
 }: Props) => {
-    const [selectedSearchScopeId, setSelectedSearchScopeId] = useState(() => preselectedProgramId);
+    const [selectedSearchScopeId, setSearchScopeId] = useState(() => (preselectedProgramId));
```

Comment:
question: why do you use a function for the initial state? is there a benefit?

Reply:
Yeah goof point. I dont know why I had it like this. I got mixed up